Makefile

```diff
@@ -53,7 +55,11 @@ all: build local-gadget
 phony_explicit:
 
 ebpf-objects:
-	TARGET=$(GOHOSTARCH) go generate ./...
+	docker run --rm --name ebpf-object-builder -v $(shell pwd):/work ghcr.io/kinvolk/inspektor-gadget-ebpf-builder
```

Comment:
You could add:
```
--user $(shell id -u):$(shell id -g)
```
so that files don't belong to root.

(as it is done in the `lint` Makefile target below)